src/bot_commands.py

```diff
@@ -646,5 +666,7 @@ def _help(
     _help,
     "List all commands"
 )
+_alias('help', 'reference')
```

Comment:
I'll be refactoring hjelpo further later, it's quite... basic right now, no?

Reply:
yeah but people will expect reference to have that output

Reply:
yeah ik this is basically just a note for later